Feedback improvements

kzimnicki-ratepay opened the pull request:
Changes: 
1. Improving README.txt - added jdbc-url.
2. Renaming ProductOrder -> Order

Commit:
upgrade and improvement the code quality.
remove lambok and map struct from the code.

Commit:
Pull request #3: upgrade and improvement the code quality.

Merge in CC/live-coding-challenge from change-product-order to main

* commit '81ffd0705df548a84cafaf8fe9fa86df96d5a4de':
  upgrade and improvement the code quality. remove lambok and map struct from the code.

dadriyanto commented:
LGTM , just one notes in the OrderControllerTest have set up 
`@Autowired
    private MockMvc mockMvc;`
    
    shall we also do the same for the ApplicationTest ?
    
    If yes then we need : 
    `@AutoConfigureMockMvc 
     @Autowired
     private MockMvc mockMvc;
`


@kzimnicki-ratepay @sadjadhoodgar

Commit:
Renaming bo->entities and remove tests methods

kzimnicki-ratepay committed:
Pull request #4: Renaming bo->entities and remove tests methods

Merge in CC/live-coding-challenge from naming-improvement to main

* commit '92b7fa195ea6c1fba69620c1568486cb5e37b9a3':
  Renaming bo->entities and remove tests methods
